testdata/repos/standalone/EclassManualDepsCheck/RubyMissingDeps/RubyMissingDeps-2.ebuild

```diff
@@ -0,0 +1,10 @@
+RUBY_OPTIONAL=1
```

Comment:
Can you add a test like:
```suggestion
RUBY_OPTIONAL=1

IUSE="test"
RESTRICT="!test? ( test )"

BDEPEND="test? ( dev-ruby/stub )"
```

to make sure this case **doesn't** warn, because it's OK?

Reply:
changed to this one.